### DISCUSSION TITLE
Are bbox results in segmentation comparable to bbox results in detection?

### DISCUSSION THREAD

**Discussion by cnut1648** [facebookresearch/detectron2#4382] [Q&A]
Hi everyone. 
I am just wondering if I have a detectron2 detection model that gets a dev bbox result A, and another detectron2 segmentation model that also gets a dev bbox result B on the same dataset, are these two results comparable? 
The reason why I ask this is that the segmentation model has additional segmentation input, while the bounding box does not. 
Thanks!

**Comment by ExtReMLapin** [comment]
I always specify area size, and bbox on both detection and segmentation training sets.

Once, I miscalculated bbox and area size on segmentation and it wouldn't train properly, I cannot be sure today if it was because of incorrect bbox or area
